Library/Formula/oauth2_proxy.rb

```diff
@@ -0,0 +1,80 @@
+require "language/go"
+
+class Oauth2Proxy < Formula
+  desc "A reverse proxy that provides authentication with Google, Github or other provider"
+  homepage "https://github.com/bitly/oauth2_proxy"
+  url "https://github.com/bitly/oauth2_proxy/archive/v2.0.1.tar.gz"
+  sha256 "febc33244d63f69a4c973e4ff2556fea2bc414308ce9979fb43db5863da87b5a"
+
+  depends_on "go" => :build
+
+  go_resource "github.com/BurntSushi/toml" do
+    url "https://github.com/BurntSushi/toml.git",
+      :revision => "056c9bc7be7190eaa7715723883caffa5f8fa3e4"
+  end
+
+  go_resource "github.com/bitly/go-simplejson" do
+    url "https://github.com/bitly/go-simplejson.git",
+      :revision => "18db6e68d8fd9cbf2e8ebe4c81a78b96fd9bf05a"
+  end
+
+  go_resource "github.com/mreiferson/go-options" do
+    url "https://github.com/mreiferson/go-options.git",
+      :revision => "7c174072188d0cfbe6f01bb457626abb22bdff52"
+  end
+
+  go_resource "gopkg.in/fsnotify.v1" do
+    url "https://gopkg.in/fsnotify.v1.git",
+      :revision => "96c060f6a6b7e0d6f75fddd10efeaca3e5d1bcb0"
+  end
+
+  def install
+    mkdir_p "#{buildpath}/src/github.com/bitly"
+    ln_s buildpath, "#{buildpath}/src/github.com/bitly/oauth2_proxy"
+
+    ENV["GOPATH"] = buildpath
+
+    Language::Go.stage_deps resources, buildpath/"src"
+    system "go", "build", "-o", "oauth2_proxy"
+    bin.install "oauth2_proxy"
+    doc.install "README.md"
+
+    (etc/"oauth2_proxy").mkpath
+    cfg = etc/"oauth2_proxy"/"oauth2_proxy.cfg"
+    if !cfg.exist?
+      touch cfg
+    end
```

Comment:
Just use `etc.install` here instead which can create the directory and install an empty file.